server/db/db.js

```diff
@@ -1,6 +1,6 @@
-const Sequelize = require("sequelize");
+const {Sequelize} = require("sequelize");
 
-const db = new Sequelize(process.env.DATABASE_URL || "postgres://localhost:5432/messenger", {
+const db = new Sequelize(process.env.DATABASE_URL || "postgres://rebecca:rebecca@localhost:5432/messenger", {
```

Comment:
It's not a good practice to have your password encoded in your code.
You can create `DATABASE_URL` variable in your `.env` file and set this string to it!

Reply:
Sure, Thanks